docs/CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 ### Changed
 
 * Make `application_root_dir` return a `Result<Path>` instead of a `String` ([#1213])
-
+* Remove unnecessary texture coordinates offset in `Sprite::from_pixel_values` ([#1267])
```

Comment:
ah, you also need to add this to the list below:

```
[#1267]: https://github.com/amethyst/amethyst/pull/1267
```

also, could you add a newline after this line

Reply:
done